forum_ng.module

```diff
@@ -128,7 +128,7 @@ function forum_ng_menu_local_tasks_alter(&$data, $router_item, $root_path) {
   global $user;
 
    // Add action link to 'node/add/forum_ng' on 'forum_ng' sub-pages.
```

Comment:
Should this comment be changed to reflect the `forums` path? e.g.

> // Add action link to 'node/add/forum_ng' on 'forums' sub-pages.

Reply:
Done.